### ISSUE TITLE
Error plotting interactions in fixed effects models (plm)

### ISSUE THREAD

**Issue by bkinne** [strengejacke/sjPlot#594]
Hello,

I'm encountering an error when trying to plot interactions from plm fit objects, specifically for models with fixed effects included. Here's an MWE with twoway fixed effects:

```r
library(plm)
library(sjPlot)
data("Produc", package="plm")
fit  <- plm(pcap ~ hwy + water + hwy*water, data=Produc, effect="twoway", index=c("state", "year"), model="within")
plot_model(fit, type="pred", terms=c("hwy", "water"))
#> Error in crossprod(beta, t(X)) : non-conformable arguments
```

I get the same error if I use only individual-level fixed effects. The error disappears and the effect plots correctly if I switch to a pooled model in the plm specification (model="pooling"), but that of course defeats the purpose of estimating a panel model.

I'm using the most recent versions of both packages.

Brandon

**Comment by strengejacke** [comment]
This looks like an issue in plm, not sjPlot.

**Comment by bckrick96** [comment]
Bump. I have same exact question. Have not been able to resolve

**Comment by tappek** [comment]
There is now support for predict for plm's within models in the development version available here: https://github.com/ycroissant/plm

As the within case is special, one likely wants to input a pdata.frame and not a plain data frame; the help page ?predict.plm has the details.

**Comment by tappek** [comment]
For models produced by `plm::plm()`, there is a `predict` method available since `plm` version 2.6-2.
